src/common/resources/exportConfigs.js

```diff
@@ -0,0 +1,25 @@
+import { get } from 'lodash';
+import { baseManifest } from '@folio/stripes-acq-components';
+
+import {
+  EXPORT_CONFIGURATIONS_API,
+} from '../constants';
+
+export const exportConfigsResource = {
+  ...baseManifest,
+  path: EXPORT_CONFIGURATIONS_API,
+  records: 'exportConfigs',
+  accumulate: true,
+  fetch: false,
+};
+
+export const credentialsResource = {
+  ...baseManifest,
+  accumulate: true,
+  fetch: false,
+  path: (queryParams, pathComponents, resourceData) => {
```

Comment:
also I'm thinking of the way not using `path` as a function, IMHO it adds extra complexity

Reply:
that much shorter and does the same job
`path: `${EXPORT_CONFIGURATIONS_API}/%{exportConfigId.id}/credentials`,`